### PR TITLE
Install Agner Fogs vector class library

### PR DESCRIPTION
**Pull request by dokempf**
The library is distributed through zip archives from the original author,
but there is a well-maintained github fork from darealshinji, which makes
the installation process easier and implementable with existing tools.

### CODE REVIEW

**Comment by dokempf** [comment]
The CE PR: https://github.com/mattgodbolt/compiler-explorer/pull/1059

File: update_compilers/install_libraries.sh [hunk 1]
```diff
@@ -288,3 +288,5 @@ get_or_sync_git_tags() {
 }
 
 get_or_sync_git_tags libs/ctre https://github.com/hanickadot/compile-time-regular-expressions.git master v2
+
+get_github_versions libs/vcl darealshinji/vectorclass v1.30
```
Comment (RabsRincon) [inline update_compilers/install_libraries.sh]: Can you move this line to Line 178? It's a better place for it

Comment (dokempf) [reply update_compilers/install_libraries.sh]: Sure! Done.

**Comment by dokempf** [comment]
Done - now for real :D

**Pull request merged**